python/iceberg/api/types/type_util.py

```diff
@@ -451,11 +451,11 @@ def write_compatibility_errors(read_schema, write_schema):
     def read_compatibility_errors(read_schema, write_schema):
         visit(write_schema, CheckCompatibility(read_schema, False))
 
-    NO_ERRORS = []
+    NO_ERRORS: List[str] = []
```

Comment:
Should this also be a tuple?

Reply:
No, it will actually return a list:
https://github.com/apache/incubator-iceberg/pull/1041/files/f466aaadac944166c24b2326575dbad47f242fc9#diff-36b662d03c587f1eb18d820c6b5f6171R504

Reply:
What I meant was: should this be `tuple()` because it is a constant and should not be mutable?